modules/check_in/spec/request/v2/patient_check_ins_request_spec.rb

```diff
@@ -63,6 +63,8 @@
       end
 
       it 'returns a success response' do
+        allow(Flipper).to receive(:enabled?)
```

Comment:
As these statements being setup procedure for the spec, can we please have these defined under before block?

Reply:
addressed in 6ea726dd366a3daf29ab012fc161ed7dd6a090f7